### ISSUE TITLE
Stack existing Convolutional Network with LSTM for Semantic pixel labelling

### ISSUE THREAD

**Issue by MehulAjax21** [fchollet/keras#6435]
Hello,
I am trying to stack the output of my convolution neural network into an LSTM. My goal is to have an end to end model trained using SGD. The output shape of my final convolutional layer is (8,224,224) using theano dimensions. After this layer, there is softmax. 
Now I want to add an LSTM between the final convolutional layer and softmax layer.
I am not sure, what kind of input shape will be for the LSTM network.

I am trying to model a reNet as per this paper:
https://arxiv.org/abs/1505.00393 

Any input or suggestion of steps are welcomed.
